UoaWDCC/AEYB
# Error Message Toast

**Closes:** [#151]

**What was done:**
Created a Toast message to show errors. Allows custom message.

## **How to test:**
- Not too sure how to test the 'server being broken' but I tested it with a button to see if it works (I've included a video below)
- Theoretically if the server gets an error, it should activate and show the toast.

**Screenshots of UI changes (optional):**
[toast.webm](https://github.com/UoaWDCC/aeyb-monitor/assets/60042992/a9dd3b3b-b6a3-4f24-9499-9a589cccabf2)
